assets/src/helpers/navIcons.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ComponentPropsWithoutRef } from "react"
+
+export type NavIconProps = ComponentPropsWithoutRef<"span">
+
+export const DetourNavIcon = (props: NavIconProps) => (
+  <span {...props}>
+    <svg
```

Comment:
suggestion: I think we should make this function like the icons in our bsIcons, so we'd ideally have the same props and we'd be putting `{...props}` on the `<svg/>` and get rid of the `<span>` inside here. If anything relies on the `<span/>` being there, I'd instead wrap this icon in some intermediate component.

If it follows our `bsicons` conventions we could potentially reuse some of that testing code as well that verifies the correct properties are present.

Reply:
I had to torture `navLinkData.ts` (now `navLinkData.tsx`) a little bit in order to get this to work.

[Thoughts?](https://github.com/mbta/skate/pull/2771/commits/3056e3e2435b323dfd8bd46bc4ca37b3bbd74862?diff=split&w=1)

Reply:
Works for me!